src/pybind/mgr/dashboard/controllers/settings.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import cherrypy
+
+from . import ApiController, RESTController, AuthRequired
+from ..settings import Settings, Options
+
+
+@ApiController('/settings')
+@AuthRequired()
+class SettingsController(RESTController):
+    @staticmethod
+    def _to_native(setting):
+        return setting.upper().replace('-', '_')
+
+    def list(self):
```

Comment:
```python
def list(self):
    return [self.get(name) for name in Options.__dict__]
```
??

Reply:
```python
return [
    json.loads(self.get(name)) for name in Options.__dict__
    if name.isupper() and not name.startswith('_')
]
```
That would work, but it requires to unserialize the result of `get` as the result it automatically serialized. Do you prefer this solution?

Reply:
why do you need to run `json.loads`?

Reply:
The `list` method of the `RESTController` class is being modified at initialization, so that it returns the serialized version of the output of `list`. I resolved this by creating a `_get` method, like you proposed in our call.